packages/documentation/docs/controls/_dropdown_styleguide.md

```diff
@@ -1,10 +1,10 @@
-Dropdown containers are elements that display a menu with additional elements as soon as a user clicks on the top element, e.g. a dropdown button. Clicking on one of the items in the dropdown triggers the action. We typically use dropdowns to allow users to select one option from a list of available choices.
+Dropdown containers are elements that display a menu with additional items as soon as a user clicks on the top element, e.g. a dropdown button. Clicking on one of the items in the dropdown triggers the action. We typically use dropdowns to allow users to select one option from a list of available choices.
 
-For information about the element that opens the dropdown container, read the article about the dropdown button [Dropdown button](dropdown-button.md) or the [Split button](split-button.md).
+For information about the element that opens the dropdown containers, please refer to our separate [Dropdown button](dropdown-button.md) and [Split button](split-button.md) guide.
```

Comment:
sounds better! I believe that it should be lowercase (dropdown and split button)
Plus, maybe "guide" is not the right word here - maybe "components"?

In general, I would stick to "dropdown" instead of "dropdown container" since this is actually the name of the component (we can discuss this in person 😉 )

Reply:
That makes sense. I followed the sentence in the style guide from the dropdown button, should we change the sentence too?